cross/toolchain.cmake

```diff
@@ -119,35 +84,6 @@ elseif(TARGET_ARCH_NAME STREQUAL "x86")
   add_compile_options(-Wno-error=unused-command-line-argument)
 endif()
 
-# Set LLDB include and library paths
-if(TARGET_ARCH_NAME MATCHES "^(arm|armel|x86)$")
-  if(TARGET_ARCH_NAME STREQUAL "x86")
-    set(LLVM_CROSS_DIR "$ENV{LLVM_CROSS_HOME}")
-  else() # arm/armel case
-    set(LLVM_CROSS_DIR "$ENV{LLVM_ARM_HOME}")
-  endif()
-  if(LLVM_CROSS_DIR)
-    set(WITH_LLDB_LIBS "${LLVM_CROSS_DIR}/lib/" CACHE STRING "")
-    set(WITH_LLDB_INCLUDES "${LLVM_CROSS_DIR}/include" CACHE STRING "")
-    set(LLDB_H "${WITH_LLDB_INCLUDES}" CACHE STRING "")
-    set(LLDB "${LLVM_CROSS_DIR}/lib/liblldb.so" CACHE STRING "")
-  else()
-    if(TARGET_ARCH_NAME STREQUAL "x86")
-      set(WITH_LLDB_LIBS "${CROSS_ROOTFS}/usr/lib/i386-linux-gnu" CACHE STRING "")
-      set(CHECK_LLVM_DIR "${CROSS_ROOTFS}/usr/lib/llvm-3.8/include")
-      if(EXISTS "${CHECK_LLVM_DIR}" AND IS_DIRECTORY "${CHECK_LLVM_DIR}")
-        set(WITH_LLDB_INCLUDES "${CHECK_LLVM_DIR}")
-      else()
-        set(WITH_LLDB_INCLUDES "${CROSS_ROOTFS}/usr/lib/llvm-3.6/include")
-      endif()
-    else() # arm/armel case
-      set(WITH_LLDB_LIBS "${CROSS_ROOTFS}/usr/lib/${TOOLCHAIN}" CACHE STRING "")
-      set(WITH_LLDB_INCLUDES "${CROSS_ROOTFS}/usr/lib/llvm-3.6/include" CACHE STRING "")
-    endif()
-  endif()
-endif()
-
-set(CMAKE_FIND_ROOT_PATH "${CROSS_ROOTFS}")
```

Comment:
Is this really not needed anymore?

Reply:
The CMAKE_SYSROOT variable also handles this.